src/com/lftechnology/java/training/alina/basic/smallestnumber/SmallestPositiveNumber.java

```diff
@@ -5,74 +5,69 @@
+        try (Scanner scanner = new Scanner(System.in)) {
+            int rangeLimit;
+            LOGGER.info("Enter the maximum range number > ");
+            rangeLimit = scanner.nextInt();
+            Long smallestNumber = displaySmallestNumber(rangeLimit);
+            LOGGER.log(Level.INFO, "The smallest positive number that is evenly divisible by all of the numbers from 1 to {0} is : {1}",
+                    new Object[] { rangeLimit, smallestNumber.toString() });
+        } catch (Exception e) {
+            LOGGER.log(Level.WARNING, "Exception Message : {0}", new Object[] { e.getMessage() });
+        }
+    }
 
-	/**
-	 * Function used to display smallest positive number
-	 * 
-	 * @param rangeLimit
-	 *            {@link Integer} maximum range
-	 * @return number {@link Long} smallest positive number
-	 * @author Alina Shakya <alinashakya@lftechnology.com>
-	 */
-	private static Long displaySmallestNumber(int rangeLimit) {
-		long number = rangeLimit;
-		int i = 2;
-		while (i < rangeLimit) {
-			number = leastCommonMultiple(number, i);
-			i++;
-		}
-		return number;
-	}
+    /**
+     * Function used to display smallest positive number
+     * 
+     * @param rangeLimit
+     *            {@link Integer} maximum range
+     * @return number {@link Long} smallest positive number
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    private static Long displaySmallestNumber(int rangeLimit) {
+        long number = rangeLimit;
+        int i = 2;
+        while (i < rangeLimit) {
+            number = leastCommonMultiple(number, i);
+            i++;
+        }
+        return number;
+    }
 
-	/**
-	 * Function to find the LCM of numbers
-	 * 
-	 * @param firstNumber
-	 * @param secondNumber
-	 * @return firstNumber
-	 * @author Alina Shakya <alinashakya@lftechnology.com>
-	 */
-	public static Long leastCommonMultiple(long firstNumber, long secondNumber) {
-		long num1, num2;
-		if (firstNumber > secondNumber) {
-			num1 = firstNumber;
-			num2 = secondNumber;
-		} else {
-			num1 = secondNumber;
-			num2 = firstNumber;
-		}
-		for (int i = 1; i <= num2; i++) {
-			if ((num1 * i) % num2 == 0) {
-				return (long) (i * num1);
-			}
-		}
-		throw new Error("Error");
-	}
+    /**
+     * Function to find the LCM of numbers
+     * 
+     * @param firstNumber
+     * @param secondNumber
+     * @return firstNumber
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static Long leastCommonMultiple(long firstNumber, long secondNumber) {
+        long num1, num2;
+        if (firstNumber > secondNumber) {
+            num1 = firstNumber;
+            num2 = secondNumber;
+        } else {
+            num1 = secondNumber;
+            num2 = firstNumber;
+        }
+        for (int i = 1; i <= num2; i++) {
+            if ((num1 * i) % num2 == 0) {
+                return (long) (i * num1);
+            }
+        }
+        throw new Error("Error");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00112)